### PR TITLE
# Added interactive list functionalities

### PR DESCRIPTION
**Pull request by direakanbi**
1. Saves completed tasks in local storage to avoid deleting upon page refresh or reload.
2. Checks out completed tasks with a checkout line.

### CODE REVIEW

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
```markdown
Hi Dire Akanbi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that gives an error and the app does not start on the `npm start` command. Please make sure you have added this command as a `scripts` in the `package.json` file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

Invalid Code Review Request **does not count** into the code reviews limit._
```

File: src/functions.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
```
Comment (Dipeshtwis) [inline src/functions.js]: - [ ] I would not recommend disabling linters, Instead, you can improve the code.

File: src/index.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
+/* eslint-disable */
```
Comment (Dipeshtwis) [inline src/index.js]: - [ ] I would not recommend disabling linters, Instead, you can improve the code.

File: src/storage.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+const checkEvent = () => {
+  const checkboxes = document.querySelectorAll('input[type=checkbox]');
+
+  for (let i = 0; i < checkboxes.length; i += 1) {
+    const task = checkboxes[i].nextElementSibling;
+
+    checkboxes[i].addEventListener('click', () => {
+      if (checkboxes[i].checked) {
+        task.style.textDecoration = 'line-through';
```
Comment (Dipeshtwis) [inline src/storage.js]: - [ ] It should have implement `change` instead of `click` event-listener on the checkbox.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @direakanbi, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- There is a warning when running the project. It's better to fix the warnings.

Screenshot from 2021-11-09 20-54-53


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Dire, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**